examples/assembling/creating_a_ken_embedding_subset.py

```diff
@@ -0,0 +1,20 @@
+"""
+.. _example_create_ken_embedding_subset:
+
+===============================
+Creating a KEN embedding subset
+===============================
+
+In this example, we will see how to use the full KEN embeddings to create
```

Comment:
Hey, thanks for this. In the discussion #578, I was more thinking of adding this to the get_ken_embeddings function description http://skrub-data.org/stable/generated/skrub.datasets.get_ken_embeddings.html, and refer to this in the KEN example.

Reply:
Thanks for the comment, I've answered over there :)